code/def_files/data/effects/main-f.sdr

```diff
@@ -293,10 +293,17 @@ void main()
  #ifdef FLAG_TEAMCOLOR
 	vec4 teamMask = vec4(0.0, 0.0, 0.0, 0.0);
 	teamMask = texture(sMiscmap, vec3(texCoord, float(sMiscmapIndex)));
-	vec3 base = max(base_color - vec3(0.5), vec3(0.0));
-	vec3 stripe = max(stripe_color - vec3(0.5), vec3(0.0));
-	baseColor.rgb += (base * teamMask.x) + (stripe * teamMask.y);
+	vec3 team_color = (base_color * teamMask.x) + (stripe_color * teamMask.y);
+	vec3 team_color_glow = (base_color * teamMask.b) + (stripe_color * teamMask.a);
+	#ifdef FLAG_HDR
+	team_color = pow(team_color, vec3(SRGB_GAMMA));
+	team_color_glow = pow(team_color_glow, vec3(SRGB_GAMMA)) * 3.0f;
```

Comment:
The last factor (`3.0`) should be explained to clarify why it is there. I assume it is for applying the glow effect to team color glows but that is not clear from the code. Also, there is a global glow color intensity value available already. Maybe that should be used here to keep everything consistent.

Reply:
It exists because of line 353:
```
 #ifdef FLAG_HDR
	glowColor = pow(glowColor, vec3(SRGB_GAMMA)) * 3.0f;
 #endif
```

Do not ask me why that exists. It just makes sense to me to apply it uniformly here. GLOW_MAP_INTENSITY is a multiplier applied to the result of everything that goes into getting the glowColor value; applying it twice seems wrong.

Reply:
I didn't know that glow maps were scaled even more in HDR mode. Thank you for clarifying this.